relayer/src/common/util.ts

```diff
@@ -0,0 +1,7 @@
+export const isUndefined = (property: unknown) => {
```

Comment:
maybe return `property is undefined` here for ts?

Reply:
it is a simple function so type inferring kicks in here. But i will add it just to be explicit.